cli/proc_state.rs

```diff
@@ -412,8 +414,6 @@ impl ProcState {
       self.prepare_node_std_graph().await?;
     }
 
-    self.progress_bar.clear();
```

Comment:
I think this might cause the progress bar to not clear when using npm packages?

Reply:
Agree, we should replace all `?` in this function with a helper that will clear the progress bar on error before bubbling up the error

Reply:
Maybe a drop handler like `let _clear_guard = pb.clear_guard();`?

Reply:
Yes, that would work and be more resilient as it will work on every code path that exits early 👍 good idea

Reply:
> I think removing this might cause the progress bar to not clear when using npm packages?

I overlooked it. probably so.

> Maybe a drop handler like let _clear_guard = pb.clear_guard();?

surely. that makes sense. :+1: